src/sass/_instance_rich_tooltip.scss

```diff
@@ -0,0 +1,29 @@
+body.is-dark .instance-rich-tooltip-table .p-icon--status-queued-small {
+  @include vf-icon-status-small($colors--light-theme--icon);
+}
+
+.instance-rich-tooltip-table .p-icon--status-queued-small {
+  @include vf-icon-status-small($colors--dark-theme--icon);
+}
+
+.instance-rich-tooltip-table {
+  margin: $sp-small 0 0;
+  min-height: 400px;
+  min-width: 350px;
```

Comment:
Can you make the width of the tooltip stable between loading and after loading? Currently, there is some flickering once it is loaded due to width change.


https://github.com/user-attachments/assets/fdfeda07-2c8d-41d1-99b6-c521c8c9470c